### PR TITLE
Disable CUDA when using FSS

### PR DESCRIPTION
**Pull request by Syzygianinfern0**
## Description
`assert not th.cuda.is_available()` makes FSS unusable on devices which have a GPU as it just exits due to the `AssertionError`. It is better to just disable the GPU and warn the user while doing so. 

## Reference
https://stackoverflow.com/a/62145307/8878627
## Checklist
- [x] I have followed the [Contribution Guidelines](https://github.com/OpenMined/.github/blob/master/CONTRIBUTING.md) and [Code of Conduct](https://github.com/OpenMined/.github/blob/master/CODE_OF_CONDUCT.md)
- [x] I have commented my code following the [OpenMined Styleguide](https://github.com/OpenMined/.github/blob/master/STYLEGUIDE.md)
- [x] I have labeled this PR with the relevant [Type labels](https://github.com/OpenMined/.github/labels?q=Type%3A)
- [ ] My changes are covered by tests

### CODE REVIEW

File: src/sympc/protocol/fss/fss.py [hunk 1]
```diff
@@ -124,7 +126,11 @@ def fss_op(x1: MPCTensor, x2: MPCTensor, op="eq") -> MPCTensor:
     Returns:
         MPCTensor: Shares of the comparison.
     """
-    assert not th.cuda.is_available()  # nosec
+    if th.cuda.is_available():
```
Comment (gmuraru) [inline src/sympc/protocol/fss/fss.py:129]: MAYBE: Save the value of `os.environ["CUDA_VISIBLE_DEVICES"]` and put it back at the end of the ```fss_op```

Comment (Syzygianinfern0) [reply src/sympc/protocol/fss/fss.py:129]: Fixed!

File: src/sympc/protocol/fss/fss.py [hunk 2]
```diff
@@ -124,7 +126,19 @@ def fss_op(x1: MPCTensor, x2: MPCTensor, op="eq") -> MPCTensor:
...
+        # FSS is currently not supported on GPU.
+        # https://stackoverflow.com/a/62145307/8878627
+        try:
+            cuda_visible_devices = os.environ["CUDA_VISIBLE_DEVICES"]
```
Comment (gmuraru) [inline src/sympc/protocol/fss/fss.py:133]: MAYBE: Use `os.environ.get("CUDA_VISIBLE_DEVICES", None)``` and remove the except block

Comment (Syzygianinfern0) [reply src/sympc/protocol/fss/fss.py:133]: Great! Never knew of it's existence.

Comment (Syzygianinfern0) [reply src/sympc/protocol/fss/fss.py]: Done! :partying_face: 
Nice tip of the `.get()`. I'll remember it :smile: :hugs:

**Review by gmuraru — APPROVED** [review]

**Pull request merged**